samples/HAProxy/haproxy.cfg

```diff
@@ -0,0 +1,46 @@
+global
```

Comment:
Where did you get this sample file? What license was it under?

Reply:
It's my own creation, using some common, but demonstration only, settings.

Reply:
The sample files are used in tests and to train the naive Bayesian classifier. We therefore try to add representative samples from "real" applications only. If you think those files are representative of common usage in `haproxy.cfg` files, that's fine by me. If not, you can replace them with any files under permissive license found on github.com.

Reply:
It's representative in that it uses many of the configuration settings that are commonly used. However, whitespace varies a lot between instances (between words). Will that cause it to not be a match? Also, the values that different projects use for each setting can vary. I've tried to use common values though. Does a line need to match in its entirety? Or only on a keyword at the beginning of the line?

Reply:
Great, thanks! Yes the Bayesian classifier acts on tokens, not on the raw files.